<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CenterDialog</class>
    <widget class="QDialog" name="CenterDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>个人中心</string>
        </property>
        <widget class="QPushButton" name="changePwdBtn">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>120</y>
                    <width>120</width>
                    <height>35</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>修改密码</string>
            </property>
        </widget>
        <widget class="QPushButton" name="changeEmailBtn">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>260</y>
                    <width>120</width>
                    <height>35</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>修改邮箱</string>
            </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowser">
            <property name="geometry">
                <rect>
                    <x>270</x>
                    <y>90</y>
                    <width>441</width>
                    <height>381</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="userNameLabel">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>40</y>
                    <width>150</width>
                    <height>50</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QPushButton" name="changeUserNameBtn">
            <property name="geometry">
                <rect>
                    <x>10</x>
                    <y>190</y>
                    <width>120</width>
                    <height>35</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>修改用户名</string>
            </property>
        </widget>
        <widget class="Line" name="line">
            <property name="geometry">
                <rect>
                    <x>160</x>
                    <y>0</y>
                    <width>3</width>
                    <height>600</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="orientation">
                <enum>Qt::Vertical</enum>
            </property>
        </widget>
        <widget class="QPushButton" name="returnBackBtn">
            <property name="geometry">
                <rect>
                    <x>640</x>
                    <y>550</y>
                    <width>120</width>
                    <height>35</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>返回</string>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>450</x>
                    <y>50</y>
                    <width>101</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>显示信息</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
